java/arcs/core/host/ResurrectorBase.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.host
+
+import arcs.core.storage.StorageKey
+import java.util.concurrent.ConcurrentHashMap
```

Comment:
This can't be used in `arcs.core`, it's part of the JDK and not part of the kotlin common stdlib.

Reply:
Any suggestions besides a mutex? Does Kotlin stdlib have any concurrent collections?

Reply:
Removed